### DISCUSSION TITLE
关于爷的信息问题

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#614] [General]
爷现在有：

>旅行者男风 旅行者女风
>旅行者男岩 旅行者女岩
>旅行者男雷 旅行者女雷

六个身份，目前好像只有风系信息，而且`信息 空`返回的头像还是荧妹。
这个命名需要规范一下，不然没办法做素材。

**Comment by mark9804** [comment]
我用的是`旅行者空` `旅行者荧`

**Reply by Xm798** [reply]
元素呢？`旅行者空风` 还是 `旅行者风空`

**Reply by mark9804** [reply]
雷荧术士听上去怪牛逼的

**Reply by Xm798** [reply]
> 雷荧术士听上去怪牛逼的

草。。。确实

**Reply by Arondight** [reply]
当初完全是为了补个没有旅行者的错误做的风荧，其实我感觉所有的旅行者都返回风莹就行了……没必要搞那么多

**Reply by Xm798** [reply]
但是命座天赋是不一样的

**Reply by Arondight** [reply]
是啊……就搞个风荧全都代替了嘛，工作量又大，而且几乎没人用，就不想做了，有个风荧意思意思，别返回个错误信息，我感觉就能糊弄一下了

**Reply by Xm798** [reply]
> 是啊……就搞个风荧全都代替了嘛，工作量又大，而且几乎没人用，就不想做了，有个风荧意思意思，别返回个错误信息，我感觉就能糊弄一下了

没人用是真的（笑

**Comment by mark9804** [comment]
其实旅行者是有一个双子的图的……返回双子就行了，信息可以说风主雷主
